kong/clustering/services/sync/rpc.lua

```diff
@@ -59,6 +55,9 @@ end
 function _M:init_cp(manager)
   local purge_delay = manager.conf.cluster_data_plane_purge_delay
 
+  -- number of versions behind before a full sync is forced
+  local FULL_SYNC_THRESHOLD = manager.conf.cluster_full_sync_threshold or 512
```

Comment:
Should not be in full capital

Reply:
Also the magic number should be named, something like "DEFAULT_SYNC_THRESHOLD"

Reply:
Added, thanks